includes/admin/upgrades/views/upgrades.php

```diff
@@ -39,81 +39,82 @@
-										esc_url( 'http://docs.givewp.com/troubleshooting-db-updates' ),
-										__( 'Read More', 'give' ) . ' &raquo;'
-									);
-
-								else:
-								?>
-									<p class="give-update-button">
-											<span class="give-doing-update-text-p" <?php echo Give_Updates::$background_updater->is_paused_process() ? 'style="display:none;"' : ''; ?>>
-												<?php
-												echo sprintf(
-													__( '%1$s <a href="%2$s" class="give-update-now %3$s">%4$s</a>', 'give' ),
-													$is_doing_updates ?
-														__( 'Give is currently updating the database in the background.', 'give' ) :
-														__( 'Give needs to update the database.', 'give' ),
-													$db_update_url,
-													( $is_doing_updates ? 'give-hidden' : '' ),
-													__( 'Update now', 'give' )
-												);
-												?>
-											</span>
-										<span class="give-update-paused-text-p" <?php echo ! Give_Updates::$background_updater->is_paused_process() ? 'style="display:none;"' : ''; ?>>
-												<?php if ( get_option( 'give_upgrade_error' ) ) : ?>
-													&nbsp;<?php _e( 'An unexpected issue occurred during the database update which caused it to stop automatically. Please contact support for assistance.', 'give' ); ?>
-												<?php else : ?>
-													<?php _e( 'The updates have been paused.', 'give' ); ?>
-
-												<?php endif; ?>
-											</span>
-
-										<?php if ( Give_Updates::$background_updater->is_paused_process() ) : ?>
-											<?php $is_disabled = isset( $_GET['give-restart-db-upgrades'] ) ? ' disabled' : ''; ?>
-											<button id="give-restart-upgrades" class="button button-primary alignright" data-redirect-url="<?php echo esc_url( admin_url( '/edit.php?post_type=give_forms&page=give-updates&give-restart-db-upgrades=1' ) ); ?>"<?php echo $is_disabled; ?>><?php _e( 'Restart Upgrades', 'give' ); ?></button>
-										<?php elseif ( $give_updates->is_doing_updates() ): ?>
-											<?php $is_disabled = isset( $_GET['give-pause-db-upgrades'] ) ? ' disabled' : ''; ?>
-											<button id="give-pause-upgrades" class="button button-primary alignright" data-redirect-url="<?php echo esc_url( admin_url( '/edit.php?post_type=give_forms&page=give-updates&give-pause-db-upgrades=1' ) ); ?>"<?php echo $is_disabled; ?>>
-												<?php _e( 'Pause Upgrades', 'give' ); ?>
-											</button>
+								<p class="give-update-button">
+									<?php
+									if( ! give_test_ajax_works() ) {
+										echo sprintf(
+											'<div class="notice notice-warning inline"><p>%s</p></div>',
+											__( 'Admin ajax is inaccesible. We will update database with Ajax request instead of backgroud, so do not leave this page after starting database updates.', 'give' )
```

Comment:
@DevinWalker Need text for this notice

![image](https://user-images.githubusercontent.com/1784821/42630053-1eb9ee08-85f3-11e8-9b62-6993e1315362.png)

Reply:
> Give is currently updating the database. Please do not refresh or leave this page while the update is in progress.